ui/src/components/useMessageDialog.tsx

```diff
@@ -97,11 +96,18 @@ const CloseButton = styled.button`
 let setMessageDialog: React.Dispatch<MessageDialogParameters>;
 let showMessageDialog: React.Dispatch<boolean>;
 
+type FormValues = {
+  message: string;
+};
+
 const MessageDialogContainer = () => {
   const messageDialogRef = React.useRef<HTMLDivElement | null>(null);
+  const textareaRef = React.useRef<HTMLTextAreaElement | null>(null);
```

Comment:
nit: textArea

Reply:
I've realised that the same functionality can be done with `key` on the textarea to force rerender, so I'll do that instead.